src/AgDatabase.cs

```diff
@@ -118,6 +118,11 @@ public void LogBackup()
       _listener.ForEachAgInstance(s => s.Restore(backupOrder, Name, retryDurationProvider, fileRelocation));
     }
 
+    public void RenameLogicalFileName(Func<string, string> fileRenamer)
+    {
+      _listener.Primary.RenameLogicalFileName(Name, fileRenamer);
+    }
+
```

Comment:
I see what we are doing, that way the caller can call this separately if they need to.

But we should be able to call this from the `Move()` function too right?
https://github.com/factset/AgDatabaseMove/blob/b3f444b7d0e85caf98a0deb9e0528067e4b4068e/src/AgDatabaseMove.cs#L54